README.md

```diff
@@ -31,6 +31,7 @@
 | Stockpile | Community crowdfunding tools for a variety of projects and initiatives | Rust, Programs, dApps | <https://github.com/StockpileProtocol/stockpile> |
 | Elusiv SDK | SDK for adding zk-privacy to dApps | TypeScript, dApps, Privacy | <https://github.com/elusiv-privacy/elusiv-sdk> |
 | Metaboss | The Metaplex NFT 'Swiss Army Knife' tool | Rust, CLI, NFTs | <https://github.com/samuelvanderwaal/metaboss> |
+| Nonci | API and SDK tool for queuing and executing transactions asynchronously using durable nonces to handle crazy bursts! | Typescript, SDK, API, Dev Tool | <https://github.com/nonci-xyz> |
```

Comment:
"Dev Tool" isn't a category, it's the whole repo

Reply:
ah removed